src/TodoForm.jsx

```diff
@@ -1,22 +1,29 @@
 import { useRef } from 'react';
+import { useState } from 'react';
 
 function TodoForm({ onAddTodo }) {
-  const todoTitleInput = useRef("");
+  const todoTitleInput = useRef('');
+  const [workingTodo, setWorkingTodo] = useState('');
```

Comment:
This was recently changed in an update to the lesson. `workingTodo` should now be `workingTodoTitle` as this more clearly conveys the kind of data this constant has.